tools/fakeroot/Makefile

```diff
@@ -0,0 +1,20 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fakeroot
+PKG_VERSION:=1.20.2
+
+PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.bz2
+PKG_SOURCE_URL:=http://snapshot.debian.org/archive/debian/20141005T221953Z/pool/main/f/fakeroot
```

Comment:
Is there a less fishy source for fakeroot available?

Reply:
We could probably use https://salsa.debian.org/clint/fakeroot instead, which is what is referred to by Debian as being the Git repo for this package.

Reply:
I would prefer that mirror as it's easier to update to future version. Updating the timestamp within the source URL seems a bad solution.

Reply:
Note that the complete source URL also includes the fakeroot version, i.e 1.20.2. The timestamp is just because snapshot.debian.org is archiving all Debian packages ever released in a timestamp based URL format.

Reply:
The GitLab repository at https://salsa.debian.org/clint/fakeroot does not seem to consistently tag releases, unless I am missing something. The version used here, 1.20.2, does not appear as a tag. This might make relying on the GitLab instance troublesome.

@tpetazzoni, have you seen tarballs archived at https://salsa.debian.org/clint/fakeroot? I could not find any.

Fedora uses https://tracker.debian.org/pkg/fakeroot as the URL for their packages, which is kind of odd.

Another option might be http://ftp.debian.org/debian/pool/main/f/fakeroot/. This is what good old freshmeat.net links to. Alpine uses http://ftp.debian.org/debian/pool/main/f/fakeroot/fakeroot_$pkgver.orig.tar.gz.

Reply:
Anything where we just have to adapt the PKG_VERSION is fine for me. Based on the git a newer version is available (not tagged). Are any of the provided sources using the latest version?

Reply:
> The GitLab repository at https://salsa.debian.org/clint/fakeroot does not seem to consistently tag releases, unless I am missing something. The version used here, 1.20.2, does not appear as a tag. This might make relying on the GitLab instance troublesome.
> 
> @tpetazzoni, have you seen tarballs archived at https://salsa.debian.org/clint/fakeroot? I could not find any.
> 
> Fedora uses https://tracker.debian.org/pkg/fakeroot as the URL for their packages, which is kind of odd.
> 
> Another option might be http://ftp.debian.org/debian/pool/main/f/fakeroot/. This is what good old freshmeat.net links to. Alpine uses http://ftp.debian.org/debian/pool/main/f/fakeroot/fakeroot_$pkgver.orig.tar.gz.

The problem with http://ftp.debian.org/debian/pool/main/f/fakeroot/ URLs is that they are not stable. Once the version is no longer used in any Debian version, the tarball disappears from there. That's why in Buildroot we use snapshot.debian.net, because URLs are guaranteed to be stable.